stable/spinnaker/templates/configmap/halyard-config.yaml

```diff
@@ -25,7 +25,7 @@ data:
     {{- end }}
     {{- end }}
 
-    $HAL_COMMAND deploy apply
+    $HAL_COMMAND deploy apply{{ if .Values.halyard.additionalInstallParameters }} {{ .Values.halyard.additionalInstallParameters }}{{ end }}
```

Comment:
```suggestion
$HAL_COMMAND deploy apply {{- range .Values.halyard.additionalInstallParameters }}
    {{ . }} \
{{ end }}
```

Reply:
something along these lines, that would output the following:
```$HAL_COMMAND deploy apply --command-a \
--command-b value.
``` 
You would probably require some work to ensure there is no trailing `\`

Reply:
I've updated my code to put each flag into its own line.

```sh
$HAL_COMMAND deploy apply \
  --command-a \
  --command-b value
```

https://github.com/helm/charts/pull/19205/files#diff-af1256557bb91bed861ac163fdf85152R29-R31

But just noticed that you suggested to put the first flag (i.e. `--command-a`) on the first line along with `deploy apply`. If you do want that style, I can update my code accordingly. Let me know.

Reply:
The way you made it looks better, thanks.